readthedocs/oauth/management/commands/sync_vcs_data.py

```diff
@@ -40,13 +40,20 @@ def add_arguments(self, parser):
             default=False,
             help='Force re-sync VCS provider data even if the users are already synced.',
         )
+        parser.add_argument(
+            '--no-dry-run',
```

Comment:
I think it's more common to use the opposite here: `--dry-run`

Reply:
Okay, I added this so that we don't accidentally trigger tasks